### ISSUE TITLE
Does not work in Thunderbird

### ISSUE THREAD

**Issue by Chesshur** [drsjb80/thematic#1]
I tried using the checkboxes and then hitting save and they WILL NOT SAVE. The checkboxes just keep resetting to unchecked everytime I run Thunderbird. In fact it does absolutely NOTHING! Using Cntrl+ALT+A does NOT work either!

I do not use Firefox instead I use Waterfox Classic which is very similar to Firefox 56 due to the fact that I despise what Mozilla had done to FF over the years. I would like to continue to using this extension, however it is a hassle having to sit and wait for WF to unhang everytime it switches themes (this happens because I have a sheer amount of themes I've collected over the years since their inception), plus the default shortcut Cntrl+Alt+A interferes with another extension so it will not work since you did not make it possible to set a custom shortcut in the options. Waterfox Classic DOES NOT have the "MANAGE EXTENSION SHORTCUTS" option since it is basically a clone of Firefox 56 so I can't do that. 

I will be using the previous version of Personas Switcher 4.0.5 until you can figure out how to get it to work with Waterfox Classic since I have no issues with switching my themes with that.

**Comment by drsjb80** [comment]
You're absolutely right, those things don't work in Thunderbird. TBird doesn't have the same interfaces as FF/WF. We do need to disable those feature checkboxes in TBird once we figure out how. We also need to document that they don't work. I'll use this issue to track those.
Can you use the PersonaSwticher preferences to change the key combination so it doesn't conflict with your other control alt a?
There aren't any plans at this point to backport the newest version to FF 56. We might be able to find time to take a look at why the performance isn't great for you. Is the auto-switching slow or is the selection from the drop-down slow?
Thanks!

**Comment by Chesshur** [comment]
It hangs on auto switch because I have a sheer amount of themes/personas I've collected over the years. I can't change the shortcut in Thematic on Waterfox since it is basically a clone of Firefox 56 and does not have the "Manage Extension Shortcuts" option like regular Firefox does now. I can only change it with Persona Switcher version 4.0.5 which does not hang. 

[Edit] After doing some investigating I've found that if I changed my version of Thunderbird back to version 68.12.1, Thematic started auto switching again and the checkboxes stayed checked. Though the  default shortcut won't work, but I can live with the it switching on its own.
